Projects/DEV/DEV03/DEV03-AWS100.md

```diff
@@ -0,0 +1,40 @@
+# DEV03-AWS100 —  Install & Configure AWS CLI then create an S3 Bucket
+
+## Cloud Service Provider
+* Amazon Web Services (AWS)
+
+## Difficulty
+Level 100 (Introductory)
+
+## Estimated Time
+* 30 - 40 minutes
+  
+## Project's Author(s)
+* [Ariela](https://twitter.com/ari_hacks)
+
+## Objectives
+
+###  You need to complete the following:
+
+* Create an IAM user (with Administrator Access) if you do not have one already
```

Comment:
Can you mention that the IAM user has to have _programmatic access_ with a pair of access keys, or create a new user with it? Someone might already have a console access user without access keys.